.travis.yml

```diff
@@ -9,7 +9,10 @@ node_js:
 install:
   - travis_retry npm install
 
-script: npm run coverage && npm run integration
+script:
+  - npm run lint
+  - npm run coverage
+  - npm run integration
```

Comment:
how about we put `npm run lint` into a `"pretest"` script or `"posttest"` script in package.json, so it automatically gets run with tests? That’s what I usually do

Reply:
Makes sense to me! I didn't realize it wasn't in there. It should definitely get run with `npm run test`.

Reply:
Looks like it is getting run with `npm test`!

Reply:
Ah I see it now. But we should probably add `npm run unit` to the list?

And move `npm run coverage` to `after_success`?

Reply:
`npm run unit`: we sure should! Hahaha.

I'm inclined to run `npm run coverage` in `after_script` rather than `after_success`. Coverage data is still useful, even if a couple unit tests fail or a lint rule fails.